storage-resize-images/functions/src/index.ts

```diff
@@ -195,24 +229,35 @@ const resizeImage = async ({
   size: string;
   objectMetadata: ObjectMetadata;
 }): Promise<ResizedImageResult> => {
-  const resizedFileName = `${fileNameWithoutExtension}_${size}${fileExtension}`;
-  // Path where resized image will be uploaded to in Storage.
-  const resizedFilePath = path.normalize(
+  const { imageType } = config;
+  let imageContentType = supportedImageContentTypeMap[imageType];
+  const hasValidImageType =
+    imageType && supportedImageContentTypeMap.hasOwnProperty(imageType);
+  if (!hasValidImageType) {
+    logs.unsupportedImageType(imageType, contentType);
```

Comment:
This logic will probably have to change. It shouldn't log out `logs.unsupportedImageType` if the user hasn't asked for a file type conversion in my opinion.

Reply:
i.e if I add a jpeg image to storage and haven't specified a file type conversion, it logs out `Image type undefined is not supported. Using image/jpeg to continue`